src/prob/mld.jl

```diff
@@ -74,6 +80,80 @@ function build_mc_mld(pm::_PM.AbstractPowerModel)
 end
 
 
+"Multinetwork load shedding problem including storage (snap-shot)"
+function build_mn_mc_mld(pm::_PM.AbstractPowerModel)
+    for (n, network) in _PM.nws(pm)
+        variable_mc_bus_voltage_indicator(pm; nw=n, relax=true)
+        variable_mc_bus_voltage_on_off(pm; nw=n)
+
+        variable_mc_branch_power(pm; nw=n)
+        variable_mc_transformer_power(pm; nw=n)
+
+        variable_mc_gen_indicator(pm; nw=n, relax=true)
+        variable_mc_gen_power_setpoint_on_off(pm; nw=n)
+
+        variable_mc_storage_power_mi(pm; nw=n, relax=true)
```

Comment:
Looking at PowerModelsRestoration, which has the most recent implimentations of mld with storage, it looks like we need here
```julia
    variable_mc_storage_indicator(pm, relax=true)
    variable_mc_storage_power_mi_on_off(pm, relax=true)
```

where the missing functions would be something like

```julia
""
function variable_mc_storage_power_mi_on_off(pm::_PM.AbstractPowerModel; relax::Bool=false, kwargs...)
    variable_mc_storage_power_real_on_off(pm; kwargs...)
    variable_mc_storage_power_imaginary_on_off(pm; kwargs...)
    variable_mc_storage_power_control_imaginary_on_off(pm; kwargs...)
    _PM.variable_storage_current(pm; kwargs...)
    _PM.variable_storage_energy(pm; kwargs...)
    _PM.variable_storage_charge(pm; kwargs...)
    _PM.variable_storage_discharge(pm; kwargs...)
    _PM.variable_storage_complementary_indicator(pm; relax=relax, kwargs...)
end
```
and
```julia
"""
a reactive power slack variable that enables the storage device to inject or
consume reactive power at its connecting bus, subject to the injection limits
of the device.
"""
function variable_mc_storage_power_control_imaginary_on_off(pm::_PM.AbstractPowerModel; nw::Int=pm.cnw, bounded::Bool=true, report::Bool=true)
    # TODO properly adapt this new control variable to multiconductor
    cnds = conductor_ids(pm; nw=nw)
    ncnds = length(cnds)

    qsc = var(pm, nw)[:qsc] = JuMP.@variable(pm.model,
        [i in ids(pm, nw, :storage)], base_name="$(nw)_qsc_$(i)",
        start = _PM.comp_start_value(ref(pm, nw, :storage, i), "qsc_start")
    )

    if bounded
        inj_lb, inj_ub = _PM.ref_calc_storage_injection_bounds(ref(pm, nw, :storage), ref(pm, nw, :bus))
        for (i,storage) in ref(pm, nw, :storage)
            if !isinf(sum(inj_lb[i])) || haskey(storage, "qmin")
                lb = max(sum(inj_lb[i]), sum(get(storage, "qmin", -Inf)))
                set_lower_bound(qsc[i], min(lb, 0.0))
            end
            if !isinf(sum(inj_ub[i])) || haskey(storage, "qmax")
                ub = min(sum(inj_ub[i]), sum(get(storage, "qmax", Inf)))
                set_upper_bound(qsc[i], max(ub, 0.0))
            end
        end
    end

    report && _IM.sol_component_value(pm, nw, :storage, :qsc, ids(pm, nw, :storage), qsc)
end
```

Reply:
I am not sure we need to go down this path.  It depends on the scope of this MLD formulation.  Best to resolve after a quick discussion.